tests/e2e/manifest/values-istio-ingress.yaml

```
@@ -0,0 +1,26 @@
+service:
+    ports:
+    ## You can add custom gateway ports in user values overrides, but it must include those ports since helm replaces.
+    # Note that AWS ELB will by default perform health checks on the first port
+    # on this list. Setting this to the health check port will ensure that health
+    # checks always work. https://github.com/istio/istio/issues/12503
+    - port: 15020
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 4

(indentation)